sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/README.md

```diff
@@ -112,7 +112,7 @@ Refer to [Logging](#logging) to enable loggers for related libraries.
 ## Next steps
 
 ### Examples
-- [./samples/event_processor_blob_storage_example.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/samples/event_processor_blob_storage_example.py) - EventHubConsumerClient with blob partition manager example
+- [./samples/event_processor_blob_storage_example.py](./samples/event_processor_blob_storage_example.py) - EventHubConsumerClient with blob partition manager example
```

Comment:
I would keep the initial link

Reply:
This should be "partition manager" - needs to be reverted to the previous "checkpoint store"

Reply:
I will revert this it must have been an bad merge.